app/assets/stylesheets/modules/landing.scss

```diff
@@ -51,7 +52,15 @@ h2 {
 }
 
 .featured-airlines {
+  h2 {
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets